scripts/provisioning/s3_start

```diff
@@ -58,6 +58,8 @@ def main():
     elif args.service == 's3bgconsumer':
       # Add action and remove pass
       pass
+    elif args.service == 'haproxy':
+      os.system("/usr/sbin/haproxy -Ws -f /etc/cortx/s3/haproxy.cfg -p /run/haproxy.pid &")
```

Comment:
should we add one isFileExists() validation for these parameters of above command command ?

Reply:
@t7ko-seagate Should we add this check suggested by @sachitanands or will provisioner do this for us?

Reply:
No I don't think we need to check for these files/params now.  Miniprovisioner must ensure they are created.